docs/source/tips_tricks.rst

```diff
@@ -32,12 +32,12 @@ Sampling Rate and Baseband Filters
 
 Using a sampling rate of less than 8MHz is not recommended. Partly, this is because the MAX5864 (ADC/DAC chip) isn't specified to operate at less than 8MHz, and therefore, no promises are made by Maxim about how it performs. But more importantly, the baseband filter in the MAX2837 has a minimum bandwidth of 1.75MHz. It can't provide enough filtering at 2MHz sampling rate to remove substantial signal energy in adjacent spectrum (more than +/-1MHz from the tuned frequency). The MAX2837 datasheet suggests that at +/-1MHz, the filter provides only 4dB attenuation, and at +/-2MHz (where a signal would alias right into the center of your 2MHz spectrum), it attenuates about 33dB. That's significant. Here's a picture:
 
-.. image:: ../images/68747470733a2f2f7261772e6769746875622e636f6d2f6d6f73736d616e6e2f6861636b72662f6d61737465722f646f632f77696b692f696d616765732f6261736562616e642d66696c7465722f6d6178323833372d316d373562772d61742d326d2e706e67.png
+.. image:: ../images/samplepicture1.png
 	:align: center
 
 At 8MHz sampling rate, and using the minimum 1.75MHz bandwidth filter, this is the response:
 
-.. image:: ../images/68747470733a2f2f7261772e6769746875622e636f6d2f6d6f73736d616e6e2f6861636b72662f6d61737465722f646f632f77696b692f696d616765732f6261736562616e642d66696c7465722f6d6178323833372d316d373562772d61742d386d2e706e67.png
+.. image:: ../images/samplepicture2.png
```

Comment:
Please rename this (and the file) to: max2837-1m75bw-at-8m.png

Reply:
Done.